test/UnbrandedProjects/Platform-OpenAI-TypeSpec/src/Generated/Models/ChatCompletionFunctions.cs

```diff
@@ -24,7 +23,7 @@ public partial class ChatCompletionFunctions
         /// `{\"type\": \"object\", \"properties\": {}}`.
         /// </param>
         /// <exception cref="ArgumentNullException"> <paramref name="name"/> or <paramref name="parameters"/> is null. </exception>
-        public ChatCompletionFunctions(string name, IDictionary<string, BinaryData> parameters)
+        public ChatCompletionFunctions(string name, ChatCompletionFunctionParameters parameters)
```

Comment:
this changes because in its [tsp definition](https://github.com/Azure/autorest.csharp/blob/5ed56117b265c61877a8c2b8b9892266071f1f0c/test/UnbrandedProjects/Platform-OpenAI-TypeSpec/completions/models.tsp#L204) we have:
```
model ChatCompletionFunctionParameters is Record<unknown>;
```
when we do not have additional properties, we parse this as a dictionary, but when we support additional properties, this should be a model with additional properties property, therefore we have this change.

Reply:
and if they only want a dictionary in their tsp, they could 
1. just use plain `Record<unknown>` on the property
2. use alias like this:
```
alias ChatCompletionFunctionParameters = Record<unknown>;
```